frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -42,6 +42,26 @@ export const GlobalStyles = (): JSX.Element => {
       --mb-color-text-medium: ${color("text-medium")};
       --mb-color-text-white: ${color("text-white")};
       --mb-color-warning: ${color("warning")};
+      --mb-color-border-alpha-30: color-mix(
+        in srgb,
+        var(--mb-color-border) 30%,
+        transparent
+      );
+      --mb-color-text-light-alpha-86: color-mix(
```

Comment:
should we follow the original name?

```suggestion
      --mb-color-text-white-alpha-86: color-mix(
```

Reply:
Whoops, didn't knew how I even missed this. Thanks!